File: Go/Easy/Remove Duplicates from LinkedList/solution_1.go
package main

type LinkedList struct {
	Value int
	Next  *LinkedList
}

func RemoveDuplicatesFromLinkedList(linkedList *LinkedList) *LinkedList {

	currentNode := linkedList

	for currentNode != nil {

		nextNode := currentNode.Next

		for nextNode != nil && nextNode.Value == currentNode.Value {

			nextNode = nextNode.Next
		}

		currentNode.Next = nextNode
		currentNode = nextNode

	}

	return linkedList

}

File: Python/Easy/Remove Duplicates From Linked List/solution_1.py
class LinkedList:
    def __init__(self,value) -> None:
        self.value = value
        self.next = None

def removeDuplicatesFromLinkedList(linkedList):

    currentNode = linkedList

    while currentNode != None:
        nextNode = currentNode.next
        while nextNode != None and  nextNode.value == currentNode.value:
            nextNode = nextNode.next
        
        currentNode.next = nextNode
        currentNode = nextNode

    return linkedList

        